lib/v2/radio-canada/latest.js

```diff
@@ -28,7 +29,9 @@ module.exports = async (ctx) => {
                     url: item.link,
                 });
 
-                item.description = detailResponse.data.match(/"body":{"html":"(.*)","attachments"/)[1].replace(/\\n/g, '');
+                const content = cheerio.load(detailResponse.data);
+                content('#main main').children('div.cTNHQH').remove();
+                item.description = content('#main main').text();
```

Comment:
```suggestion
                const rcState = detailResponse.data
                    .match(/JSON\.parse\("(.*?)"\);/)[1]
                    .replace(/\\"/g, '"')
                    .replace(/\\\\"/g, '\\"');
                const rcStateJson = JSON.parse(rcState);
                const news = Object.values(rcStateJson.pagesV2.pages)[0];
                item.description = news.data.newsStory.body.html.replace(/\\n/g, '<br>');
```
https://docs.rsshub.app/joinus/new-rss/before-start#acquire-data
Embedded JSON is perfered over article HTML.

Reply:
Thanks. The regex is somehow complex  and not really understand how it is done by using these 'replace' function. Got to learn new staff from your maintainers.